### DISCUSSION TITLE
Problem trying to redeploy a previously terminated Hubs Cloud

### DISCUSSION THREAD

**Discussion by TechThomas** [mozilla/hubs#5482] [Hubs Cloud Questions]
Hey, having problem with trying to get my hubs cloud to regenerate after I had to shut it down.  I think I have old data left over from a previous deploy.  Tried many things and information I found on the Bug DB.  My current error is "The following resource(s) failed to create: [AppCloudfrontDistribution, LinkRedirectorCloudfrontDistribution]."

Are there any steps for how to completely wipe my EC2 / Route 53 information to make a deployment go successfully? 

<img width="1141" alt="HubIssue" src="https://user-images.githubusercontent.com/991176/171875678-d5f3036b-74b5-4f08-920e-ff4c3d18d855.png">

**Comment by pattersonbl2** [comment]
if you manually deleted resources from your previous stack,  you would have to find everything that stack created and delete by hand. 

this is a resource on deleting resources: https://aws.amazon.com/premiumsupport/knowledge-center/terminate-resources-account-closure/
